book/basics.qmd

```diff
@@ -17,22 +17,35 @@ abstract: TODO (150-200 WORDS)
 Examples of such learners include classification tree learners (`r ref("mlr_learners_classif.rpart", text = "classif.rpart")`), regression support vector machine learners (`r ref("mlr_learners_regr.svm", text = "regr.svm")`), and many others, see `r link("https://mlr-org.com/learners.html", "the complete list here")`.
-The model a learner constructs is saved in the [learner](#sec-learners) object and can then used to produce [predictions](#sec-predicting) on the test data.
-These predictions can be compared to the ground truth values to assess the quality of the model with various [performance measures](#sec-measure).
-Usually, the value of a [measure](#sec-measure) is a numeric score.
-This value is usually called the estimate of the generalization error -- given new data that we have not seen before, how well do we estimate the learned model to perform?
-
-Partitioning the entire data set into training and test sets is called [resampling](#resampling) in `r mlr3`.
-A single resampling may not provide the best estimate of the generalization performance because it is based on only a single data point.
-As data are usually partitioned randomly, a single split can produce training and test sets that are very different, hence creating the misleading impression that the particular type of model does not perform well.
-Repeating the procedure of partitioning, building a model on the training set, and evaluating it on the test set gives multiple such data points and in general provides a more robust estimate of the generalization performance.
-
+Many machine learning algorithms make use of the concept of \emph{empirical risk minimization} during [training](#sec-training) to produce their fitted model i.e., they optimize a loss function on the training data over all candidate models.
+A loss function quantifies the mismatch between target values as predicted by a model and ground truth values as found in the data.
+This empirical risk is only a stochastic proxy for the theoretical risk or true generalization error.
+
+In `r mlr3`, the model constructed by a learner during training is directly saved in the learner object and can be used to produce [predictions](#sec-predicting) on the test data.
+As we are interested in models that generalize beyond the training data rather than just memorizing it, separate test data allows to evaluate models in an unbiased way and assess to what extent they have learned the relationships that underlie the data.
+[Evaluating](#sec-eval) a model is done based on a performance measure, usually again based on a loss function.
+This value is often called the estimate of the generalization error -- given new data that we have not seen before, how well do we estimate the learned model to perform?
+In general, the loss function used during training, i.e., empirical risk minimization must not be the same loss function on which the performance measure is based on.
```

Comment:
Bit confusing, sounds like you're saying a model can't be tuned with RMSE and then evaluated with RMSE.